test/e2e/layers/layerAttribute.test.js

```diff
@@ -0,0 +1,45 @@
+import { test, expect, chromium } from '@playwright/test';
+
+test.describe("Adding Opacity Attribute to the Layer- Element", () => {
+  let page;
+  let context;
+  test.beforeAll(async function() {
+    context = await chromium.launchPersistentContext('');
+    page = context.pages().find((page) => page.url() === 'about:blank') || await context.newPage();
+    await page.goto("layerAttribute.html");
+  });
+  test.afterAll(async function () {
+    await context.close();
+  });
+
+  test("Setting Opacity Attibute to Layer- Element", async () => {
+    let opacity_attribute_value = await page.$eval("body > mapml-viewer > layer-",(layer) => layer.getAttribute("opacity"));
+    if (opacity_attribute_value == null){
```

Comment:
I believe you want to use "===" mostly in JavaScript.  I believe that you can re-phrase the logic of this as:

```javascript
if (opacity_attribute_value) {
...
}